relayer/src/channel.rs

```diff
@@ -24,7 +24,22 @@ use crate::connection::Connection;
 use crate::error::Error;
 use crate::foreign_client::{ForeignClient, ForeignClientError};
 
-const MAX_RETRIES: usize = 5;
+use crate::util::retry::{count_retry, retry_with_index};
+
+mod retry_strategy {
+    use crate::util::retry::clamp_total;
+    use retry::delay::Fibonacci;
+    use std::time::Duration;
+
+    // Default parameters for the retrying mechanism
+    const MAX_DELAY: Duration = Duration::from_secs(60); // 1 minute
+    const MAX_TOTAL_DELAY: Duration = Duration::from_secs(10 * 60); // 10 minutes
+    const INITIAL_DELAY: Duration = Duration::from_secs(1); // 1 second
```

Comment:
I am making a copy of `retry_strategy` from `monitor.rs`, as I suspect we may need to change the delay parameters to a smaller duration?

Reply:
Yes for now it's better to have each module define its own retry strategy until we figure out exactly what are the best parameters. At some we will maybe want to update those dynamically depending on the error rate, block time, etc.